src/GUI/GamePanel.java

```diff
@@ -0,0 +1,79 @@
+package GUI;
+
+import Gameplay.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.IOException;
+
+public class GamePanel extends JPanel {
+
+    protected static int XOffset = 100;
```

Comment:
Any reason for this to be `protected`?
```suggestion
    private static int XOffset = 100;
```